test-e2e/tests/account_test.ts

```diff
@@ -10,35 +10,27 @@ const firstNameField = 'firstName';
 const lastNameField = 'lastName';
 const consentCheckbox = 'Yes, I want to receive Blender updates by email';
 
-let loginContext: LoginContext;
+const loginContexts: { [key: string]: LoginContext } = {};
```

Comment:
Does this need to be a dictionary? I would assume the tests run in isolated instances and could just both use a single value here still.